lib/inspec/control_eval_context.rb

```diff
@@ -40,7 +40,7 @@ def method_missing(method_name, *arguments, &block)
           if hook
             # OK, load the hook if it hasn't been already.  We'll then know a module,
             # which we can then inject into the context
-            registry.activate(:control_dsl, method_name) unless hook.activated?
+            hook.activate unless hook.activated?
```

Comment:
Any harm in simplifying all the `hook.activate unless hook.activated?` to be only `hook.activate`?

The `activate()` method has [a quick `return if activated?` already present](https://github.com/inspec/inspec/pull/3720/files#diff-80fde85ee69a12f9aad186be59ae05c3R23).

Reply:
A fine idea, sir.